src/authentication/middleware/index.ts

```diff
@@ -18,20 +18,22 @@ export default (req: Request, res: Response, next: NextFunction) => {
   }
 
   logger.debug("Check if user is signed in");
-  if (!req.chSession.isSignedIn()) {
+  if (!req.originalUrl.endsWith(pageURLs.ACCESSIBILITY_STATEMENT)) {
+    if (!req.chSession.isSignedIn()) {
```

Comment:
could these two not combined into a single OR operation?
If (notSignedIn OR accessingAccessibilityStatement)...

Reply:
Ignore this comment it wouldn't work.